enrichment-web/src/main/resources/config/enrichment.properties

```diff
@@ -59,4 +59,4 @@ enrich.items.import=C:\\Users\\katicd\\Documents\\Europeana\\Code\\Ait\\addition
 
 # saving Wikidata info into files
 enrich.wikidata.directory = C:/wikidata

```

Comment:
we should have a system independent predefined path something like /app/enrichment/wikidata

Reply:
Yes, improved.